frontend/src/pages/Dashboard.vue

```diff
@@ -67,22 +68,27 @@ let categorySearched = reactive({
   category: null,
 });
 let isLoading = ref(true);
+let showList = ref(true);
 
 const handleSearch = (val, title) => {
   if (title === undefined) {
     categorySearched.category = "RANDOM";
     apiSearched.value = apiData.value;
+    showList.value = true;
   } else if (val.length > 0) {
     categorySearched.category = title.value.toUpperCase();
     apiSearched.value = val;
+    showList.value = true;
   } else {
     categorySearched.category = "NO MATCH";
     apiSearched.value = apiData.value;
```

Comment:
Well if we remove this line then we may not get result if we clear one word from input. 

for eg:- when we try to search 'NBAA' the result get 'NOT MATCH' but if we clear one words as 'NBA" its should be display list since the input is valid. because we are into this condition, so it need to be update value of 'showList'.